secret-sharing/src/suites/p384.rs

```diff
@@ -21,7 +17,7 @@ impl GroupDigest for Sha3_384 {
 
     fn hash_to_group(msg: &[u8], dst: &[u8]) -> Result<Self::Output> {
         let msgs = [msg];
-        let dsts = [NIST_P384_SHA3_384_ENC_DST, dst];
+        let dsts = [dst];
```

Comment:
Did you also consider accepting a slice of DSTs or is there a reason against it?

Reply:
No, could also use that. The only reason against was to simplify the interface, as I cannot return `dst: &[&[u8]]` from helper method `domain_separation_tag`. I need to create a vector, copy data, and pass the slice, so accepting slices of DSTs is not needed.

Reply:
Ok, leave it as-is.

Reply:
Plus, the method doesn't behave like `TupleHash`, so dsts `[[1,2],[3]]` and `[[1],[2,3]]` are the same, which could deceive the caller.